packages/story-editor/src/components/publishModal/content/mainStoryInfo.js

```diff
@@ -35,14 +35,18 @@ import { INPUT_KEYS } from '../constants';
 
 const FormSection = styled.div`
   padding: 0 4px;
-  margin: 18px 0 8px;
   &:first-of-type {
-    margin-top: 20px;
+    margin: 20px 0 0px;
```

Comment:
nit: `margin: 20px 0 0;` or `margin: 20px 0px 0px;`

I think the codebase does the unit-less version more for `0` values, but I'm fine with either

Reply:
oof I didn't even notice I did that 🤦🏻‍♀️ Thanks.